src/components/carousel/carousel.api.json

```diff
@@ -22,7 +22,8 @@
     {
       "name": "containerMarginHorizontal",
       "type": "number",
-      "description": "Horizontal margin for the carousel container"
+      "description": "Horizontal margin for the carousel container",
+      "default": "true"
```

Comment:
The type is `number` and the default value is `true`?

Reply:
haha. It' of the 'horizontal' prop. Fixed